python/cuml/common/base.pyx

```diff
@@ -59,8 +59,8 @@ class Base:
         algo = MyAlgo(handle=handle)
         algo.fit(...)
         result = algo.predict(...)
-        # final synchronization of all work launched/dependent on this stream
-        stream.sync()
+        # final sync of all gpu-work launched inside this object
```

Comment:
Might be worth adding to the comment that this is equivalent to `stream.sync()` in this case, but `handle.sync()` is the saver equivalent in no custom stream is created to be used by `handle`, i.e. the default stream is used.

Reply:
Done